scripts/setup-adapters.sh

```diff
@@ -22,13 +22,85 @@ set -eufx -o pipefail
 SCRIPTDIR=$(dirname "${BASH_SOURCE[0]}")
 source $SCRIPTDIR/setup-helper-functions.sh
 DEPENDENCY_DIR=${DEPENDENCY_DIR:-$(pwd)}
+INSTALL_PREFIX="${INSTALL_PREFIX:=${DEPENDENCY_DIR}/install}"
```

Comment:
Is this needed? Why not stick with `DEPENDENCY_DIR`? We can then use `scripts/setup-helper-functions.sh` and the `github_checkout` function below. Also, why deviate from github_checkout and use github_wget_and_untar?

Reply:
I was previously using the instructions to install the dependencies of GCS as specified by Google, but you are right, there was no need to deviate. Fixed.